app/src/main/java/edu/mit/mitmobile2/tour/fragment/TourFragment.java

```diff
@@ -0,0 +1,161 @@
+package edu.mit.mitmobile2.tour.fragment;
+
+import android.app.AlertDialog;
+import android.app.Fragment;
+import android.content.ActivityNotFoundException;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.List;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+
+import edu.mit.mitmobile2.Constants;
+import edu.mit.mitmobile2.MITAPIClient;
+import edu.mit.mitmobile2.R;
+import edu.mit.mitmobile2.tour.model.MITTourWrapper;
+import edu.mit.mitmobile2.tour.utils.TourUtils;
+import retrofit.Callback;
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+
+public class TourFragment extends Fragment {
+
+    int contentLayoutId = R.layout.content_tours;
+
+    @InjectView(R.id.self_guided_tour_view)
+    RelativeLayout selfGuidedTourView;
+    @InjectView(R.id.send_feedback_view)
+    RelativeLayout sendFeedbackView;
+    @InjectView(R.id.mit_info_center_view)
+    RelativeLayout mitInfoCenterView;
+    @InjectView(R.id.mit_admissions_view)
+    RelativeLayout mitAdmissionsView;
+    @InjectView(R.id.more_guided_tours_text_view)
+    TextView moreGuidedToursTextView;
+    @InjectView(R.id.more_about_mit_text_view)
+    TextView moreAboutMitTextView;
+    @InjectView(R.id.self_guided_tour_title)
+    TextView selfGuidedTourTitleTextView;
+    @InjectView(R.id.stops_info_text_view)
+    TextView stopsInfoTextView;
+    @InjectView(R.id.distance_info_text_view)
+    TextView distanceInfoTextView;
+    @InjectView(R.id.time_info_text_view)
+    TextView timeInfoTextView;
+
+    private MITAPIClient mitapiClient;
+
+    public TourFragment() {
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.content_tours, null);
+
+        ButterKnife.inject(this, view);
+
+        mitapiClient = new MITAPIClient(getActivity().getApplicationContext());
+
+        mitapiClient.get(Constants.TOURS, Constants.Tours.TOUR_PATH, null, null, new Callback<List<MITTourWrapper>>() {
+            @Override
+            public void success(List<MITTourWrapper> mitTourWrappers, Response response) {
+                MITTourWrapper mitTourWrapper = mitTourWrappers.get(0);
+                setTourInfoView(mitTourWrapper);
+            }
+
+            @Override
+            public void failure(RetrofitError error) {
+            }
+        });
+
+        selfGuidedTourView.setBackground(getResources().getDrawable(R.drawable.tours_cover_image));
+
+        moreGuidedToursTextView.setOnClickListener(new View.OnClickListener() {
```

Comment:
If you're going to use Butterknife to inject views, then you should also use it for click handling, for the sake of consistency